deploy/concierge/rbac.yaml

```diff
@@ -67,6 +61,36 @@ roleRef:
   name: #@ defaultResourceNameWithSuffix("aggregated-api-server")
   apiGroup: rbac.authorization.k8s.io
 
+#! Give permission to the kube-cert-agent Pod to run privileged.
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: #@ defaultResourceNameWithSuffix("kube-cert-agent")
```

Comment:
I would not expect this to be specific to the cert agent - same comment applies to the binding.